app/assets/javascripts/kuroko2/job_instances.js

```diff
@@ -45,6 +48,7 @@ jQuery(function ($) {
         tbody.data('last-log-id', data.logs[data.logs.length - 1].id);
       }
 
+      $('#logs table').data("reload", data.reload);
```

Comment:
Okay, but I couldn't figure out why and how this is related to "Prevent notifying each time" ...